assignments/mini-libc/samples/buffered_io.c

```diff
@@ -0,0 +1,24 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+
+int main(void)
+{
+	FILE *f;
```

Comment:
It throws a compilation error.
Where is `FILE` structure defined?

Reply:
Typically it's in `bits/types/FILE.h`. For convenience, I think it can be defined in `stdio.h`.

Reply:
This will be removed. It's a lingering artifact.